app/pages/cif/projects.tsx

```diff
@@ -57,30 +65,39 @@ export const ProjectsQuery = graphql`
         }
       }
     }
+    allCifUsers {
+      edges {
+        node {
+          fullName
+        }
+      }
+    }
   }
 `;
 
-const tableFilters = [
-  new TextFilter("Project Name", "projectName"),
-  new TextFilter("Operator Trade Name", "operatorTradeName", {
-    orderByPrefix: "OPERATOR_BY_OPERATOR_ID__TRADE_NAME",
-  }),
-  new TextFilter("Proposal Reference", "proposalReference"),
-  new TextFilter("Status", "status", {
-    orderByPrefix: "PROJECT_STATUS_BY_PROJECT_STATUS_ID__NAME",
-  }),
-  new DisplayOnlyFilter("Assigned To"),
-  new SortOnlyFilter("Funding Request", "totalFundingRequest"),
-  new NoHeaderFilter(),
-];
-
 export function Projects({ preloadedQuery }: RelayProps<{}, projectsQuery>) {
-  const { allProjects, pendingNewProjectRevision, session } = usePreloadedQuery(
-    ProjectsQuery,
-    preloadedQuery
-  );
+  const { allProjects, allCifUsers, pendingNewProjectRevision, session } =
+    usePreloadedQuery(ProjectsQuery, preloadedQuery);
   const router = useRouter();
 
+  const tableFilters = [
+    new TextFilter("Project Name", "projectName"),
+    new TextFilter("Operator Trade Name", "operatorTradeName", {
+      orderByPrefix: "OPERATOR_BY_OPERATOR_ID__TRADE_NAME",
+    }),
+    new TextFilter("Proposal Reference", "proposalReference"),
+    new TextFilter("Status", "status", {
+      orderByPrefix: "PROJECT_STATUS_BY_PROJECT_STATUS_ID__NAME",
+    }),
+    new SearchableDropdownFilter(
+      "Assigned To",
```

Comment:
I recall some confusion about this being "Assigned To", while the project form says "Managers" should we rename the header?